dcpy/utils/logging.py

```diff
@@ -1,6 +1,23 @@
 import logging
 
-logging.basicConfig()
+
+class LoggingHandler(logging.StreamHandler):
+    def __init__(self):
+        super(LoggingHandler, self).__init__()
+
+    def emit(self, record):
+        # Ensure multi-line logs have consistent log formatting
+        messages = record.msg.split("\n")
+        for message in messages:
+            record.msg = message
+            super(LoggingHandler, self).emit(record)
+
+
 logging.root.setLevel(logging.INFO)
 
+handler = LoggingHandler()
+formatter = logging.Formatter(logging.BASIC_FORMAT)
+handler.setFormatter(formatter)
+
 logger = logging.getLogger("dcpy")
+logger.addHandler(handler)
```

Comment:
Got an example/link to how this ends up looking?

Reply:
oops I added this for since-removed new python code in KPDB. just rebased to actually use it in a Template DB step, will link to how this looks asap

Reply:
example in logs [here](https://github.com/NYCPlanning/data-engineering/actions/runs/7401969645/job/20138858341?pr=240#step:10:18)